ui/apps/platform/src/hooks/useFetchDeploymentCount.ts

```diff
@@ -30,18 +31,10 @@ function useFetchDeploymentCount(selectedClusterId: string): UseFetchDeploymentC
         queryOptions
     );
 
-    useEffect(() => {
-        if (!data || !data.count) {
-            return;
-        }
-
-        setDeploymentCount(data.count);
-    }, [data]);
-
     return {
         loading,
         error,
-        deploymentCount,
+        deploymentCount: data?.cluster?.count || undefined,
```

Comment:
Is the OK to be undefined, or does it need to default to 0 ?

Reply:
From the type it looked like it's either a `number` or `undefined` so I tried to keep it as expected. I believe it previously would be `undefined` until we set a number from the `useState<number>()`.